pylint/pyreverse/diagrams.py

```diff
@@ -304,3 +310,10 @@ def extract_relationships(self) -> None:
                 except KeyError:
                     continue
                 self.add_relationship(package_obj, dep, "depends")
+
+            for dep_name in package_obj.node.type_depends:
+                try:
+                    dep = self.get_module(dep_name, package_obj.node)
+                except KeyError:
+                    continue
```

Comment:
These two lines are uncovered, and I don't know how to cover them. Any ideas? They could also be cut, or just ignore the coverage.